ghc-events-analyze.cabal

```diff
@@ -58,9 +58,9 @@ executable ghc-events-analyze
                        diagrams-lib         >= 1.3   && < 1.5,
                        diagrams-svg         >= 1.1   && < 1.5,
                        filepath             >= 1.3   && < 1.5,
-                       ghc-events           >= 0.13  && < 0.14,
+                       ghc-events           >= 0.13  && < 0.17,
                        hashable             >= 1.2   && < 1.4,
-                       lens                 >= 3.10  && < 4.20,
+                       lens                 >= 3.10  && < 5.0,
```

Comment:
`5 < 5.0`, and there is `lens-5`.

Reply:
(it looks like you cannot get `lens-5` due `diagrams` doesn't allowing `lens-5`, but that may change at any moment.